mpf/crawler.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from pyexcel_ods import get_data 
+from pyexcel_xls import get_data 
+import pathlib 
+
+#Processo de download dos dados do MPF
+def consulta(year,month,data_type,output_path):
+    base_url = 'http://www.transparencia.mpf.mp.br/conteudo/contracheque/'+ data_type +'/'
+   
+    #Não trabalha com determinados caracteres
+    if(month == 'Março'):
+        month = 'Marco'
+
+    #O formato dos arquivos (extension) muda para .ods a partir de Junho de 2019 
+    extension = '.xls'
+    if(int(year) == 2020):
+        extension = '.ods'
+    elif((int(year) == 2019) and ( (month == 'Junho') or (month == "Julho") or (month == "Agosto") or (month == "Setembro") or (month == "Outubro") or (month == "Novembro") or (month == 'Dezembro'))):
```

Comment:
Acho que aqui a gente podia simplificar, criar um array de meses e colocar elif month in ARRAY...